### DISCUSSION TITLE
Commas not showing on page load.

### DISCUSSION THREAD

**Discussion by mathewGlenn** [dm4t2/vue-currency-input#362] [Q&A]
I am using Nuxt3, and I created a reusable component of the vue-currency-input. I put in in a page, and when I load the page, a value should display on the currency input. The value does show by using v-model, however, the commas do not show until I click on the input.

This is my code:

  const options = {
  currency: 'EUR',
  currencyDisplay: 'hidden',
  precision: 2,
  hideCurrencySymbolOnFocus: true,
  hideGroupingSeparatorOnFocus: false,
  hideNegligibleDecimalDigitsOnFocus: true,
  autoDecimalDigits: false,
  useGrouping: true,
  accountingSign: false,
};

 <MetaCurrencyInput
 v-model.lazy="state.form.number_of_applications"
:options="options"
:disabled="false"
@keyup="compute('noa')"
/>

**Comment by dm4t2** [comment]
Your issue is not described clearly. What means commas? Can you share a minimal reproduction?
